integration_tests/test_local_plugin.py

```diff
@@ -28,7 +28,8 @@ def test_stage_local_plugin(self):
         self.run_snapcraft('stage', project_dir)
 
         self.assertThat(
-            os.path.join(project_dir, 'stage', 'build-stamp'), FileExists())
+            os.path.join(project_dir, 'stage', 'build-stamp'),
+            FileExists())
```

Comment:
Can't this fit on one line? (I figure static checks would have caught it otherwise)